constants.ts

```diff
@@ -0,0 +1,40 @@
+import { Chain } from "./types";
+
+export const chains: Chain[] = [
```

Comment:
what's the point for this? wouldn't be easier to just maybe import these chains from our `@thirdweb-dev/chains` package?

Reply:
this array is used to [render the select](https://github.com/thirdweb-example/thirdweb-livepeer/blob/763d2edf7ad5bb2f948ce5d68a021bc65caf98e2/components/shared/Select.tsx#L24-L28) so user can select which chain their token/contract is deployed. 

can you please share the code for `@thirdweb-dev/chains`, will check if it is possible

Reply:
`import { Goerli, Polygon } from "@thirdweb-dev/chains"`